uwblueprint/ccbc
# Add Creator Publications Section

## Notion ticket link

[Booker: Creator Publications Section](https://www.notion.so/uwblueprintexecs/Booker-Creator-Publications-Section-1e249b17e19b461784f19fff57d85d27?pvs=4)



## Implementation description
* Added CreatorPublications component, to be displayed on the Creator Overview Page
* Displays the first 3 publications of the creator with publication name, publisher name, description and book cover image
* View all link that routes user to the magazine review platform



## Steps to test
1. View a Creator Profile that contains publications
2. Ensure that the first 3 publications of the creator are shown with the publication name, publisher name, description and book cover image
3. Check to see if the view all button redirects to the magazine review platform with the correct creator



## What should reviewers focus on?
* Styling of images, sometimes the images get squished when resizing the page


## Checklist
